Library/Formula/clang-omp.rb

```diff
@@ -15,19 +16,35 @@ class ClangOmp < Formula
   depends_on "libiomp"
   depends_on "cmake" => :build
 
-  resource "compiler-rt" do
-    url "https://github.com/clang-omp/compiler-rt/archive/2015-04-01.tar.gz"
-    sha256 "5a8d39ff6ce524e23fae32870f85b18d43f2795da2011d3cbb6b29d471bb27b7"
-  end
+  stable do
```

Comment:
Whoops, fixing now.  I was under the false impression those were shared, or 'global'.